java/example.conf

```diff
@@ -0,0 +1,32 @@
+# This is an example ray config file.
+# To use this file, copy it to your classpath and rename it to 'ray.conf'.
+
+# For all available config items, see 'java/runtime/src/main/resources/ray.default.conf'.
+# For config file format, see 'https://github.com/lightbend/config/blob/master/HOCON.md'.
+
+ray {
+
+  // Path of your Ray home dir,
+  // can be either an absolute path, or a relative path to current work dir.
+  home = "/path/to/your/ray/home"
```

Comment:
Is this the `ray/` directory where Ray is installed? E.g., if I clone the Ray repository to `/home/ubuntu/ray` and then build it from source in place, then this should be `/home/ubuntu/ray`?

"current working directory" sounds like something else, e.g., that sounds more like where my application code lives. So maybe this comment can be clarified?

Reply:
> Is this the ray/ directory where Ray is installed?

yes.

> "current working directory" sounds like something else, e.g., that sounds more like where my application code lives. So maybe this comment can be clarified?

The motivation of allowing 'relative path to current work dir' is the following: when we run java tests under `ray/java/test` folder, ray home will be `../..` of current work dir, see `TestListener.java`. If we only allow absolute path, everyone needs to change it before running tests.

Reply:
I see, then can we rephrase the comment as

```
// This is the path to the directory where Ray is installed, e.g.,
// something like /home/ubuntu/ray. This can be an absolute path
// or a relative path from the current working directory.
```

To clarify, is it a relative path from the directory that the user is in? Or it a relative path from where the application lives?